codelabs/keyboard_navigation/keyboard_navigation.md

```diff
@@ -411,13 +392,45 @@ CustomCursor.prototype.onBlocklyAction = function(action) {
       var rootBlock = currentBlock.getRootBlock();
       // Gets the top node on a block. This is either the previous connection,
       // output connection, or the block itself.
-      var topNode = Blockly.navigation.getTopNode(rootBlock);
+      var topNode = Blockly.ASTNode.createTopNode(rootBlock);
       // Update the location of the cursor.
-      this.setCurNode(topNode);
+      cursor.setCurNode(topNode);
+      return true;
     }
   }
 };
 ```
+Once we have created the shortcut, we can now register it. Add the below code
+to `playground.html` in order to register your shortcut.
+```js
+Blockly.ShortcutRegistry.registry.register(moveToStack);
+```
+
+### Adding a key mapping
+Now that we have registered our shortcut, we can add key mappings. A key mapping is
+a mapping from a key code or combination of key codes to a shortcut. When the
+key code or combination of key codes have been pressed the shortcut will run.
+
+We can combine primary keys with modifier keys by using the `createSerializedKey`
+method. A list of the available modifier keys are:
+1. `Blockly.ShortcutRegistry.modifierKeys.SHIFT`
+1. `Blockly.ShortcutRegistry.modifierKeys.CONTROL`
+1. `Blockly.ShortcutRegistry.modifierKeys.ALT`
+1. `Blockly.ShortcutRegistry.modifierKeys.META`
+
+For our example we will create a key code for control W by using the below code.
```

Comment:
Add instructions to user to add the following code as well.

Reply:
done